book/02-git-basics/sections/recording-changes.asc

```diff
@@ -212,24 +212,24 @@ $ cat .gitignore
 *~
 ----
 
-The first line tells Git to ignore any files ending in ``.o'' or ``.a'' -- object and archive files that may be the product of building your code.
-The second line tells Git to ignore all files whose names end with a tilde (`~`), which is used by many text editors such as Emacs to mark temporary files.
-You may also include a log, tmp, or pid directory; automatically generated documentation; and so on.
-Setting up a `.gitignore` file for your new repository before you get going is generally a good idea so you don't accidentally commit files that you really don't want in your Git repository.
+Ang unang linya ay nagsasabi ng Git na hindi pansinin ang anumang mga file na nagtatapos sa ``.o'' o ``.a'' -- na bagay at archive na mga file na maaari ang produkto ng pag-build ng iyong code.
```

Comment:
I think the word `object` is a file type here. Hence, should not be translated.

Reply:
one change.